check/check.go

```diff
@@ -161,8 +162,11 @@ func (c *Check) Run() {
 		glog.V(2).Info(errmsgs)
 	}
 
-	res := c.Tests.Execute(out.String())
-	if res {
+	// Todo: check result for nil
+	finalOutput := c.Tests.Execute(out.String())
```

Comment:
It's not a big deal, but I'm not sure I would have combined the flag and the string into a structure here, as we immediately take them out of it again. Would have been more succinct to have something like 
```
    res, c.ActualValue := c.Tests.Execute(out.String())
    if res {
    ...
```